### PR TITLE
Add Cyryl to the website

### PR DESCRIPTION
**Pull request by cyplo**

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -216,3 +216,14 @@
     <p>Now it's time to bring software craftmanship to Swift mobile, desktop and server.</p>
   twitter: guillemf
   github: guillemf
+
+- id: cyplo
+  name: Cyryl Płotnicki
+  image: /assets/img/custom/team/cyryl_plotnicki.jpg
+  role: Software Craftsman
+  fullDescription:
+    <p> Cyryl spent most of his professional life building and maintaining larger backend systems, mostly in .Net and JVM spaces. Gradually he became interested in the people management side of the work as well. Via this interest he found the software crafstmanship community and never looked back. He is focused on extracting the needs of the clients and helping them get the value out of the work quickly. </p>
```
Comment (jukecraft) [inline _data/team.yml]: Cyryl spent most of his professional life building and maintaining larger backend systems, mainly in .Net and JVM ecosystems. Gradually he became interested in the people management side of the work as well. Via this interest he found the software craftsmanship community and never looked back. He is focused on extracting clients' needs and helping them to get value out of the work quickly.

Comment (ghost) [inline _data/team.yml]: maybe you mean "large backend systems"

File: _data/team.yml [hunk 2]
```diff
@@ -216,3 +216,14 @@
...
+    <p> Interested making the systems, whether physical or virtual, more secure. Open source advocate with strong data privacy focus. He tries to try at least one new programming language a year, focusing on Rust lately. </p>
```
Comment (ghost) [inline _data/team.yml]: interested -in- making the systems..

Comment (jukecraft) [inline _data/team.yml]: Being an open source advocate with strong data privacy focus, he's passionate about making systems more secure, whether they are physical or virtual. He tries to learn at least one new programming language every year, with the latest being Rust.

Comment (cyplo) [reply _data/team.yml]: good catch, thank you

File: _data/team.yml [hunk 3]
```diff
@@ -216,3 +216,14 @@
...
+    <p> Outside of work Cyryl likes to relax by taking part in the local hackerspace community, making electronics and machine parts.</p>
```
Comment (jukecraft) [inline _data/team.yml]: Outside of work, Cyryl likes to relax by taking part in the local hackerspace community, making electronics and machine parts.

**Review by jukecraft — COMMENTED** [review]
minor grammatical suggestions

File: _data/team.yml (hunk 1 above)
Comment (cyplo) [reply _data/team.yml]: Lovely, thank you :)

Comment (cyplo) [reply _data/team.yml]: I think they are just larger, would not call them large I think ;)

**Review by jukecraft — APPROVED** [review]

**Pull request merged**